src/app/@ansyn/overlays/models/base-overlay-source-provider.model.ts

```diff
@@ -26,6 +27,7 @@ export interface IFetchParams {
 	sensors?: string[];
 	dataInputFilters: IDataInputFilterValue[];
 	timeRange: IDateRange;
+	worldName?: string;
```

Comment:
Cancel. Do not change IFetchParams. Instead, make a new interface inside your provider, which will extend IFetchParams